### PR TITLE
Add section for alternative implementation of mendeleev to README

### PR DESCRIPTION
**Pull request by lmmentel**
- add rust implementation

### CODE REVIEW

**Comment by lmmentel** [comment]
@YgorSousa anything to add?

File: README.md [hunk 1]
```diff
@@ -488,6 +488,12 @@ Feel free to submit [issues](https://github.com/lmmentel/mendeleev/issues) regar
 -   submit a [pull request](https://github.com/lmmentel/mendeleev/pulls)
     so that your changes can be reviewed
 
+## Other languages
+
+Due to great work of other people there are aleternative version in different languages:
```
Comment (YgorSouza) [inline README.md:493]: There's just a typo here in "alternative", but otherwise I think that's fine.

**Comment by lmmentel** [comment]
Should we also add the link to the github repo?

**Comment by YgorSouza** [comment]
> Should we also add the link to the github repo?

It's on GitLab, actually. There is a link to the repo on the crate page, and there is a badge with a link to the crate in the repo's readme, so either way is fine I think. If you want the badge:

```
[![Latest version](https://img.shields.io/crates/v/mendeleev.svg)](https://crates.io/crates/mendeleev)
```

[Latest version](https://crates.io/crates/mendeleev)

**Review by YgorSouza — APPROVED** [review]

**Pull request merged**